compose.yaml

```diff
@@ -34,11 +35,16 @@ services:
       POSTGRES_USER: cashflow
       POSTGRES_PASSWORD: cashflow
       POSTGRES_DB: cashflow
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
```

Comment:
I don't see a reason to add this as data is persisted as long as you don't run `docker compose down` and it hides the data for those who already have something in their local databases

Reply:
I didn't know and wanted to be sure that the fake data I inserted for tests purposes would be persistent.